archunit/src/main/java/com/tngtech/archunit/library/Architectures.java

```diff
@@ -426,6 +438,12 @@ public OnionArchitecture adapter(String name, String... packageIdentifiers) {
             return this;
         }
 
+        @PublicAPI(usage = ACCESS)
+        public OnionArchitecture allowEmptyLayers(boolean allowEmptyLayers) {
```

Comment:
I wonder if `allowEmptyLayers` is the right expression, when we switch the context to `onionArchitecture`. Because the whole point of the Onion Architecture infrastructure was pretty much to provide better naming than "layers". Unfortunately I don't have any spontaneous ideas for better wording :thinking: What is a common name for the parts of the Onion Architecture?

Reply:
I believe that [The Onion Architecture](https://jeffreypalermo.com/2008/07/the-onion-architecture-part-1/) still consists of _"layers"_ – even though the architecture as a whole is considered a replacement for a _layered_ architecture.

Reply:
Hmm, I guess so, in the end an onion kind of has "layers" :wink: